examples/demo/wmr.config.js

```diff
@@ -0,0 +1,7 @@
+function wmr_config () {
+	return {
+		alias: {
+			'src/*': 'src'
+		}
+	};
+}export default wmr_config;
```

Comment:
This file doesn't look like it belongs to this PR.

Reply:
Thank you 🙏